pyodide_build/recipe/builder.py

```diff
@@ -176,7 +177,8 @@ def build(self) -> None:
         try:
             self._build()
 
-            (self.build_dir / ".packaged").touch()
+            if not self.is_wheel:
+                (self.build_dir / ".packaged").touch()
```

Comment:
I think it could be improved by implementing it in each child class. You don't need to implement it in this PR, and since I was working on subclassing, I can do it all at once in another PR.